scripts/show-features.py

```diff
@@ -122,6 +122,50 @@ def main(argv=None):
         else:
             functions = filter(lambda f: f.va == args.function, functions)
 
+            if args.function not in [f.va for f in functions]:
```

Comment:
using a set here might be slightly faster: `{f.va for f in functions}`. 

or with shortcircuit: `if any(map(lambda f: f.va == args.function, functions))`

but it works just fine as is :-) not in the hot path